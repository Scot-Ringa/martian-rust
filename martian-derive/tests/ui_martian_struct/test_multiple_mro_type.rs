use martian_derive::MartianStruct;

struct Foo(f32);

#[derive(MartianStruct)]
struct InvalidField {
    num_reads: i64,
    #[mro_type = "int"]
    #[mro_type = "float"] // Cannot specify twice
    foo: Foo,
}

fn main() {}
