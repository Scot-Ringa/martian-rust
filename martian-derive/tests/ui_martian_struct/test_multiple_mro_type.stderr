error: Looks like you are setting #[mro_type] twice for field 'foo'
  --> $DIR/test_multiple_mro_type.rs:8:5
   |
8  | /     #[mro_type = "int"]
9  | |     #[mro_type = "float"] // Cannot specify twice
10 | |     foo: Foo,
   | |____________^
